configure.ac

```diff
@@ -372,23 +372,6 @@ case $CSPREFIX in
   prefix) CSPREFIX=$prefix;;
 esac
 
-AC_MSG_NOTICE(Sanitizing INITDIR: ${INITDIR})
-case $INITDIR in
-  prefix) INITDIR=$prefix;;
-  "")
-    AC_MSG_CHECKING(which init (rc) directory to use)
-      for initdir in /etc/init.d /etc/rc.d/init.d /sbin/init.d	\
-	   /usr/local/etc/rc.d /etc/rc.d
```

Comment:
One of the point of configure script is to provide interface alleviating the build system from second-guessing of how to result should look like with regards to the target system.  Doing so based on basically random existence of particular directories is a perfect example of "overly smart second-guessing".  When building in a minimalistic chroot (such as with RPM ecosystem using mock heavily), they may not even exist at all.  So if very basic `/etc/init.d` wvon't fit, (fairly advanced by definition) user is supposed to specify the location via `--with-initdir=DIR`.

What's the conflict?  What the surprises are there when help screen navigates one to do so?

If you cannot follow these intuitive steps, you should delegate the building question to someone else, really.